Proposed changes to right pane look&feel

zthomae opened the pull request:
This is more of a request for comment than anything else, because I don't know if you want to go in this direction.

I've (temporarily?) added simplified column width classes to restore some semblance of how the grids used to look, because (IMO) it's a lot easier to iterate by preserving an existing layout with a visual aid.

I reverted some changes to move grid widths into element-specific styles because I think those stragglers would be confusing given the above.

Summary of the changes:
- Set overflow to auto for the tab contents.
- Remove right border on tab buttons container.
- Fix the table layout of the player roster (with column classes, for now).
- Align the text in the inventory sections vertically and horizontally.
- Make item stack background color darker grey, to differ from the border.
- Add border at the bottom of the inventory.
- Give the map a slightly larger minimum height.
- Remove horizontal rules. They were ugly before and are nonfunctioning now.

Screenshots:

![roster](https://user-images.githubusercontent.com/884552/37074336-ed9d6d42-2191-11e8-845e-c8374766d5c2.png)
![game information](https://user-images.githubusercontent.com/884552/37074340-efc45d1a-2191-11e8-937d-dfd53a8aacf2.png)

zthomae committed:
Remove horizontal rules

They were ugly before and are nonfunctional now

zthomae committed:
Add simplified bootstrap grid classes for migration

This might be a hot take, but I've done this to restore some semblance
of how the grids used to look, because (IMO) it's a lot easier to
iterate by preserving an existing layout with a visual aid.

I reverted some changes to move grid widths into element-specific
styles because I think those stragglers would be confusing given the
above.

zthomae committed:
Remove right border on tab contents

The frame is already providing a border here.

zthomae committed:
Add overflow: auto to tab-contents

zthomae committed:
Add left-right padding inside roster table

This makes some room for the vertical lines marking the current player
in the roster.

zthomae committed:
Improve styling of the player inventory and map

This regularizes the positioning of the text in the table, as well as
changes the background color of the item stacks for better contrast.

A minimum height is set for the map, to give it a bit of breathing
room. Perhaps a maximum height is also in order here?

zthomae committed:
Increase padding around roster rows

LOZORD commented:
All those UI changes seem good to me. Thanks!
Please fix the broken test; then I'll go through, nitpick as necessary, and then submit.

zthomae committed:
Fix test for gold appearing in the view

I removed the colon from the text when I made the appearance of the
gold and the inventory box more consistent, and forgot to update the
test.

LOZORD commented on assets/client/index.html:
```diff
@@ -65,26 +65,26 @@ <h2 id='player-info-class'>class name</h2>
 				<div id='tab-contents'>
 					<div id='roster' class='tab-content'>
-						<div id='roster-header'>
+						<div id='roster-header' class='col-xs-12'>
```
Since we are no longer constrained by the dogma of Bootstrap, you can remove the `xs` here. `col-12`, `w12`, `sp12`, etc. is fine with me.

zthomae replied:
I'd like to keep the grid as it was previously defined working, and I think renaming those classes would make this PR a lot noisier. I think we can revisit this later.

LOZORD commented on assets/client/index.html:
```diff
@@ -176,22 +176,20 @@ <h3>Effects</h3>
-								<div class='col-xs-9'>
-									<h4 id='gold-amount'>gold here</h4>
+							<div id='gold-row'>
+								<div class='row'>
```
Why did you separate these divs?

zthomae replied:
Good question.

LOZORD reviewed (approved):
Thanks again!

zthomae committed:
Remove unnecessary div wrapping

LOZORD committed:
Add TODO about col-xs prefix.

LOZORD committed:
Comment cleanup

LOZORD commented:
We still have to get rid of all of the jQuery, but that can be in a different PR.
